backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/incremental/IncrementalLayoutEngine.java

```diff
@@ -14,14 +14,23 @@
 
 import java.math.BigDecimal;
 import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.Optional;
 
+import javax.annotation.Nonnull;
```

Comment:
Why would that be necessary?

Reply:
If you want to tell the consumer the returned value can be null you can use Optional

Reply:
I removed it.